### PR TITLE
Use rosinstall file to gather deps for travis build.

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
@130s: I was under the impression that adding `UPSTREAM_WORKSPACE=file` to the entries under `matrix` and adding a `.travis.rosinstall` file would be enough to get Travis to checkout additional packages before starting the build. Is that not correct?

Looking at [the raw log](https://s3.amazonaws.com/archive.travis-ci.org/jobs/143587305/log.txt) of the build for this PR, searching for `UPSTREAM_WORKSPACE` finds it three times:
- the first one `exports` the value I specified (ie: `file`)
- the second one seems to override it again to `debian`
- the third then checks what the value is (`debian`) and decides to "Obtain deb binary for upstream packages", which fails (as `staubli` has not been released)

Would I need to do something else to get this to work?

**Comment by 130s** [comment]
What you've done sounds enough and the result sounds strange. I'll look.

File: .travis.yml [hunk 1]
```diff
@@ -14,9 +14,11 @@ env:
     - USE_DEB=true
```
Comment (130s) [inline .travis.yml]: If you want to use `UPSTREAM_WORKSPACE=file`, then `USE_DEB` needs to be either `false` or simple not defined.

File: .travis.yml [hunk 2]
```diff
@@ -14,9 +14,11 @@ env:
...
       ROS_DISTRO="indigo"
       ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+      UPSTREAM_WORKSPACE=file
     - USE_DEB=true
```
Comment (130s) [inline .travis.yml]: Same as [the comment above](https://github.com/ros-industrial/staubli_experimental/pull/12/files#r70192123).

**Comment by 130s** [comment]
Found a cause and left comments.

I'll also update readme to clarify this point. Thanks!

**Comment by gavanderhoorn** [comment]
Ah, right. That makes sense actually. Should've paid more attention.

Thanks for taking a look.

**Comment by gavanderhoorn** [comment]
Thanks @130s for clearing things up. Travis now works with the additional package present.

**Pull request merged**